test/v1/models/audit/CreateLossClaimAuditDetailSpec.scala

```diff
@@ -28,55 +29,76 @@ class CreateLossClaimAuditDetailSpec extends UnitSpec {
     "work" when {
       "success response" in {
         val json = Json.parse(s"""{
-             |    "userType": "Agent",
-             |    "agentReferenceNumber":"012345678",
-             |    "nino": "$nino",
-             |    "request": {
-             |      "typeOfLoss":"self-employment",
-             |      "selfEmploymentId":"X2IS12356589871",
-             |      "typeOfClaim":"carry-forward",
-             |      "taxYear": "2019-20"
-             |    },
-             |    "response":{
-             |      "httpStatus": 201,
-             |      "body":{
-             |        "id": "$claimId",
-             |        "links": [{
-             |          "href": "/individuals/losses/$nino/loss-claims/$claimId",
-             |          "method": "GET",
-             |          "rel": "self"
-             |        }
-             |        ]
-             |     }
-             |    },
-             |    "X-CorrelationId": "a1e8057e-fbbc-47a8-a8b478d9f015c253"
-             |}
-           """.stripMargin)
+            |    "userType": "Agent",
+            |    "agentReferenceNumber":"012345678",
+            |    "nino": "$nino",
+            |    "request": {
+            |      "typeOfLoss":"self-employment",
+            |      "selfEmploymentId":"X2IS12356589871",
+            |      "typeOfClaim":"carry-forward",
+            |      "taxYear": "2019-20"
+            |    },
+            |    "response":{
+            |      "httpStatus": 201,
+            |      "body":{
+            |        "id": "$claimId",
+            |        "links":[
+            |        {
+            |            "href":"/individuals/losses/{$nino}/loss-claims/{$claimId}",
+            |            "rel":"self",
+            |            "method":"GET"
+            |        },
+            |       {
```

Comment:
It's debatable, but not sure there's a point in having all links here, though as the JSON is arbitrary and (a) it might obscure what is being tested and (b) do we really want to have to keep this up to date if links change. 

I was toying with the idea of just having a foo/bar completely arbitrary request and response JSON.

???

Reply:
It doesn't really add any value to have the response and request be accurate the endpoint so that's probably a good idea. It'll clean up the specs a bit as well.